lua/entities/golfrack.lua

```diff
@@ -0,0 +1,84 @@
+if SERVER then AddCSLuaFile() end
+ENT.Type = "anim"
+DEFINE_BASECLASS("base_gmodentity")
+ENT.Model = Model("models/swamponions/golfstand.mdl")
+
+function ENT:Initialize()
+    self:SetModel(self.Model)
+    self:SetMoveType(MOVETYPE_NONE)
+	self:PhysicsInitStatic(SOLID_VPHYSICS)
+
+    self:DrawShadow(false)
+	
+    self:SetSolid(SOLID_VPHYSICS)
+
+	local phys = self:GetPhysicsObject()
+	if IsValid(phys) then
+        phys:EnableMotion(false)
+	end
+end
+
+function ENT:Draw()
+    self:DrawModel()
+end
+
+function ENT:Think()
+    if SERVER then
+        for k, v in pairs(ents.FindInSphere(self:GetPos(), 60)) do
```

Comment:
This is way too expensive. Can you do it with entity:Touch or :Use instead?

Reply:
Sure. That's why I put in a NextThink(), I figured it would be performance-heavy